bin/curriculum/backfill-allow-multiple-attempts.rb

```diff
@@ -18,7 +18,7 @@ def level_is_standalone?(level)
 end
 
 def level_is_contained?(level)
-  level.levels_parent_levels.count {|l| l.kind == 'contained'} > 0
+  ParentLevelsChildLevel.where(child_level_id: level.id).where(kind: 'contained').count > 0
```

Comment:
The `levels_parent_levels` association wasn't always working locally for some reason. I made this query more explicit to ensure I caught all contained levels.